test/net/sourceforge/kolmafia/persistence/FaxBotDatabaseTest.java

```diff
@@ -172,27 +173,31 @@ public void checkPreferenceWhenNoFiles() {
       // different.
       globalCleanup.close();
       FaxBotDatabase.resetInitialization();
+      FileUtilities.setTestingWithFakeData(true);
       var cleanups = new Cleanups(withNextResponse(200, response), withProperty(property, false));
       try (cleanups) {
         assertFalse(getBoolean(property));
         FaxBotDatabase.configure();
         // This test fails because the ResettingHttpClient in FileUtilities does not actually get
         // the fake response.
-        // assertTrue(getBoolean(property));
+        assertTrue(getBoolean(property));
       }
+      FileUtilities.setTestingWithFakeData(false);
     }
 
     @Test
     public void checkPreferenceWhenFiles() {
       String property = "_faxDataChanged";
       FaxBotDatabase.resetInitialization();
-      var cleanups = new Cleanups(withNextResponse(200, response), withProperty(property, false));
+      FileUtilities.setTestingWithFakeData(true);
+      var cleanups = new Cleanups(withNextResponse(200, ""), withProperty(property, false));
       try (cleanups) {
         assertFalse(getBoolean(property));
         FaxBotDatabase.configure();
         // Not sure why not working
-        // assertTrue(getBoolean(property));
+        assertTrue(getBoolean(property));
       }
+      FileUtilities.setTestingWithFakeData(false);
```

Comment:
also this set / unset should be in a try / finally as if the test fails the unset won't get called

Reply:
I think you are saying that if I implement your previous suggestion correctly this will not be needed and the test alone would perform the function.